### PR TITLE
Add liveness checks to the controller-manager and scheduler.

### PR DESCRIPTION
**Pull request by brendandburns**

### CODE REVIEW

File: cluster/saltbase/salt/kube-scheduler/kube-scheduler.manifest [hunk 1]
```diff
@@ -13,6 +13,14 @@
                  "-c",
                  "/usr/local/bin/kube-scheduler --master=127.0.0.1:8080 {{pillar['log_level']}} 1>>/var/log/kube-scheduler.log 2>&1"
                ],
+    "livenessProbe": {
+      "httpGet": {
+        "path": "/healthz",
+        "port": 10251
+      },
+      "initialDelaySeconds": 15,
+      "timeoutSeconds": 1
```
Comment (lavalamp) [inline cluster/saltbase/salt/kube-scheduler/kube-scheduler.manifest]: Really? Every second? That seems excessive. How about every 10 seconds?

Comment (brendandburns) [inline cluster/saltbase/salt/kube-scheduler/kube-scheduler.manifest]: done.

**Comment by brendandburns** [comment]
Comment addressed, ptal.

Thanks!
--brendan

**Comment by lavalamp** [comment]
LGTM

**Pull request merged**